ntbs-service/Pages/Reports/Index.cshtml.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Configuration;
+
+namespace ntbs_service
+{
+    public class IndexModel : PageModel
```

Comment:
Let's add a comment that explains why we want a thing controller like this in the first place, rather than linking directly out